src/main/java/com/intuit/graphql/orchestrator/schema/fold/UnifiedXtextGraphFolder.java

```diff
@@ -103,7 +115,7 @@ private UnifiedXtextGraph merge(UnifiedXtextGraph accumulator, XtextGraph curren
           );
     }
 
-    resolveTypeConflicts(accumulator.getTypes(), current.getTypes(), current);
+    resolveTypeConflicts(accumulator.getTypes(), current.getTypes(), accumulator, current);
```

Comment:
can we just pass the accumulator and current to this method ?

Reply:
make sense will do

Reply:
done